docs/get-started.md

```diff
@@ -4,14 +4,43 @@ title: Temporal documentation
 sidebar_label: Get started
 ---
 
+Welcome to Temporal's user documentation!
+
 Temporal reimagines state-dependent service-orchestrated application development, and so much more.
 
 ## Get started
 
-This is page will be dedicated to directing users on how to get started with Temporal.
+Ready to develop? [Install the Temporal server](install-temporal-server) and choose your SDK.
```

Comment:
I like the tone change A LOT. I'm not necessarily saying we should change the word "develop" but not sure its what we want to communicate.

Reply:
Very unsure here, just my initial thought.

Reply:
What about "Ready to build?"